contracts/LifCrowdsale.sol

```diff
@@ -238,13 +231,17 @@ contract LifCrowdsale is Ownable, Pausable {
 
      @param beneficiary Address to which Lif will be sent
      @param weiSent Amount of wei contributed
+     @param rate of tokens per ether at the moment of the contribution
```

Comment:
I'd change it to:

`@param rate Lif per ether rate at the moment of the contribution`

(changed tokens to Lif, and also note that rate is the param name, and the rest is the description of the param, it's not supposed to be a regular phrase as a whole but first the param, then the description)

Reply:
:+1: